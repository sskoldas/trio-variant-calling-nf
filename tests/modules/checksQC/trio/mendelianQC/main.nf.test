nextflow_process {

    name "Test Process TRIO_MENDELIAN_QC"
    script "modules/checksQC/trio/mendelianQC/main.nf"
    process "TRIO_MENDELIAN_QC"

    test("Should run without failures") {

        when {
            params {
                outdir = "tests/results"
            }
            process {
                """
                input[0] = file("${projectDir}/tests/modules/checksQC/trio/mendelianQC/input/mock.table")
                input[1] = "mock_cohort"
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out.mendel).match()
        }

    }

}
