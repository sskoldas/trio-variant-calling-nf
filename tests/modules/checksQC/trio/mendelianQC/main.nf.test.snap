{
    "Should run without failures": {
        "content": [
            [
                "mock_cohort.mendelian_qc.txt:md5,0367cd983311a6684df4cb1bb6f37704"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.6"
        },
        "timestamp": "2025-09-28T23:38:22.540055228"
    }
}